docs/settings.rst

```diff
@@ -168,6 +168,15 @@ Setting name (followed by default value, if any)
                                                                                  is also in ``ARTICLE_PATHS`` or ``PAGE_PATHS``. If you are trying
                                                                                  to publish your site's source files, consider using the
                                                                                  ``OUTPUT_SOURCES`` setting instead.
+``STATIC_CREATE_LINKS = False``                                                  Create links instead of copying files. If the content and output
```

Comment:
I remember why I didn't call it `STATIC_CREATE_HARDLINKS` in the first place. That's because if the content and output directories are on different filesystems, then the program will fallback to creating symbolic links so I picked a more generic name.